openlibrary/templates/type/edition/view.html

```diff
@@ -38,6 +38,11 @@
 $ edition_count = work.edition_count if work and work.edition_count else 1
 $ meta_cover_url = item_image((edition or work).get_cover_url("L"), default="https://openlibrary.org/images/icons/avatar_book-sm.png")
 $ _x = ctx.setdefault('bodyid', 'book')
+$ base_url = "https://openlibrary.org"
```

Comment:
I _think_ this should work with `ctx.homedomain` ? But you might want to test it first.